packages/common/src/enums.ts

```diff
@@ -21,6 +21,7 @@ export enum Hardfork {
   Berlin = 'berlin',
   London = 'london',
   ArrowGlacier = 'arrowGlacier',
+  GrayGlacier = 'grayGlacier',
```

Comment:
Unrelated, but since you are already working on the code: can you please move `Shanghai` to the last position here? This is some relic from times where people thought Shanghai would happen before the Merge. 😋

(general note: please do all change requests on both PRs, maybe do all in one commit and then cherry-pick over?)